README.md

```diff
@@ -23,4 +23,11 @@ export let ApiConfig = {
 ```
 yarn
 ng serve
-```
\ No newline at end of file
+```
+
+## Testing Development Tips
```

Comment:
Suggestion:
<pre><code>To isolate a spec file per run, change the function name `it` to `fit` or `describe` to `fdescribe`.
To speed up compilation, turn off sourcemaps:
```bash
ng test --sourcemap=false
```
</code></pre>